packages/stable/src/__tests__/api/views.int.spec.ts

```diff
@@ -0,0 +1,175 @@
+// Copyright 2024 Cognite AS
+
+import { ViewCreateDefinition } from '../../api/views/types.gen';
+import CogniteClient from '../../cogniteClient';
+import { setupLoggedInClient } from '../testUtils';
+
+describe('Views integration test', () => {
+  let client: CogniteClient;
+  const timestamp = Date.now();
+
+  const TEST_SPACE_NAME = `Views_integration_test_${timestamp}`;
+  const TEST_CONTAINER_NAME = `Views_integration_test_container${timestamp}`;
+
+  const viewCreationDefinition: ViewCreateDefinition = {
+    externalId: 'test_view',
+    space: TEST_SPACE_NAME,
+    name: 'test_data_space',
+    description: 'Instance space used for integration tests.',
+    version: '1',
+    properties: {
+      test: {
+        container: {
+          type: 'container',
+          externalId: TEST_CONTAINER_NAME,
+          space: TEST_SPACE_NAME,
+        },
+        containerPropertyIdentifier: 'test',
+      },
+    },
+  };
+  const viewCreationDefinition2: ViewCreateDefinition = {
+    externalId: 'test_view_2',
+    space: TEST_SPACE_NAME,
+    name: 'test_data_space_2',
+    description: 'Instance space used for integration tests.',
+    version: '1',
+    properties: {
+      test: {
+        container: {
+          type: 'container',
+          externalId: TEST_CONTAINER_NAME,
+          space: TEST_SPACE_NAME,
+        },
+        containerPropertyIdentifier: 'test',
+      },
+    },
+  };
+
+  beforeAll(async () => {
+    client = setupLoggedInClient();
+    await client.spaces.upsert([
+      {
+        space: TEST_SPACE_NAME,
+        name: TEST_SPACE_NAME,
+        description: 'Instance space used for views integration tests.',
+      },
+    ]);
+    await client.containers.upsert([
+      {
+        externalId: TEST_CONTAINER_NAME,
+        space: TEST_SPACE_NAME,
+        name: TEST_SPACE_NAME,
+        description: 'Instance space used for views integration tests.',
+        properties: {
+          test: {
+            type: { type: 'text' },
+          },
+        },
+      },
+    ]);
+  });
+  afterAll(async () => {
+    client = setupLoggedInClient();
+    await client.spaces.delete([TEST_SPACE_NAME]);
```

Comment:
Might have to delete containers before spaces, ref this comment: https://cognitedata.slack.com/archives/C03G11UNHBJ/p1706885816266049?thread_ts=1706884545.460149&cid=C03G11UNHBJ